js/spotify.js

```diff
@@ -1,12 +1,43 @@
-// API Docs at: 
+// API Docs at:
 // https://developer.spotify.com/web-api/search-item/
+$(document).ready(function () {
+  $('form').on('submit', function (e) {
+    e.preventDefault();
+  var keyword = $('#search-keyword').val();
+  searchByArtist();
+  })
+
+
+
 
 
 function searchByArtist(keyword) {
-  var url = 'https://api.spotify.com/v1/search?q='+keyword+'&type=artist';
-}
+     var url = 'https://api.spotify.com/v1/search?q='+keyword+'&type=artist';
+     var request = $.ajax(url);
+
+     request.done(function (response) {
+       data = response;
+       $artist = response.artists;
+       console.log($artist);
+       var artistList = $('<li />').html($artist);
+       $('#results').append(artistList);
+    });
+     request.fail(function (jqx, status, errorThrown) {
+      console.log("error: ", jqx, status, errorThrown);
+    });
+     request.always(function () {
```

Comment:
You don't even have to include this `.always` function if you're not doing anything inside of it. Your console.log of nothing doesn't help you any here so I'd just delete it entirely or change what's being consoled.